frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/namespace-store.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { commonFlows } from '../views/common';
+import { store, Providers, testName, StoreType } from '../views/store';
+import { checkErrors } from '../../../integration-tests-cypress/support';
+
+describe('Tests creation of Namespace Stores', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+    store.setStoreType(StoreType.NamespaceStore);
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  afterEach(() => {
+    cy.byLegacyTestID('actions-menu-button').click();
+    cy.byTestActionID('Delete Namespace Store').click();
+    cy.byTestID('confirm-action').click();
+    checkErrors();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    commonFlows.navigateToOCS();
+    cy.byLegacyTestID('horizontal-link-Namespace Store').click();
+    cy.byTestID('item-create').click();
+  });
+
+  it('Test creation of AWS namespace store', () => {
+    store.createStore(Providers.AWS);
+    cy.byLegacyTestID('resource-title').contains(testName);
```

Comment:
I think you also need to check the state of namespace store, verifying its status, etc.